pkg/integrations/v4/logs/cfg.go

```diff
@@ -170,6 +170,7 @@ type FBCfgInput struct {
 	DB                    string
 	Path                  string // plugin: tail
 	BufferMaxSize         string // plugin: tail
+	Path_Key	      string // plugin: tail
 	SkipLongLines         string // always on
 	Systemd_Filter        string // plugin: systemd
```

Comment:
```suggestion
	SystemdFilter        string // plugin: systemd
```
Just noticed this one as well.

Reply:
Systemd_Filter is not part of the code I changed/added.  Not sure if it's appropriate for me to change it.